src/main/resources/config/application-artemis.yml

```
@@ -21,6 +21,10 @@ artemis:
         internal-admin:
             username: artemis_admin
             password: artemis_admin
+        registration:
+            enabled: false
+            allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@((tum\.de)|(in\.tum\.de)|(mytum\.de))
```

We should be able to do that as well:
```suggestion
            allowed-email-pattern: [a-zA-Z0-9_\-\.\+]+@(tum\.de|in\.tum\.de|mytum\.de)
```

---

Checked it, also works like this.

---

thanks, this looks easier, I'll change it